src/Akeneo/Pim/Enrichment/Component/Product/Normalizer/Standard/Product/ProductValueNormalizer.php

```diff
@@ -128,4 +126,26 @@ protected function getSimpleValue(ValueInterface $value, ?string $format = null,
 
         return $this->normalizer->normalize($value->getData(), $format, $context);
     }
+
+    /**
+     * Cut all the ending zeros after coma if decimals are more than 4
+     * @return int|string
+     */
+    private function formatNumber(ValueInterface $value, AttributeInterface $attribute)
+    {
+        if (!$attribute->isDecimalsAllowed()) {
+            return (int)$value->getData();
+        }
+
+        $dataWithoutZero = preg_replace('/0+$/', '', $value->getData());
+
+        $valueExploded = explode(".", $dataWithoutZero);
+        $valueBehindComa = array_pop($valueExploded);
+
+        if (strlen($valueBehindComa) < 4) {
+            return number_format($dataWithoutZero, static::DECIMAL_PRECISION, '.', ',');
```

Comment:
```suggestion
            return number_format($dataWithoutZero, static::DECIMAL_PRECISION, '.', '');
```
I think we should not have any thousands separator